integration-auth/src/main/java/org/symphonyoss/integration/authentication/AuthenticationContext.java

```diff
@@ -50,15 +66,47 @@
 
   public AuthenticationContext(String userId, KeyStore keyStore, String keyStorePass) {
     this.userId = userId;
+    this.client = buildClient(keyStore, keyStorePass);;
+  }
 
+  private Client buildClient(KeyStore keyStore, String keyStorePass) {
     final ClientConfig clientConfig = new ClientConfig();
     clientConfig.register(MultiPartFeature.class);
 
-    final ClientBuilder clientBuilder = ClientBuilder.newBuilder()
-        .keyStore(keyStore, keyStorePass)
-        .withConfig(clientConfig);
+    // Connect and read timeouts in milliseconds
+    clientConfig.property(ClientProperties.READ_TIMEOUT, READ_TIMEOUT);
+    clientConfig.property(ClientProperties.CONNECT_TIMEOUT, CONNECT_TIMEOUT);
+
+    // Socket factory setup with custom SSL context settings
+    SSLConnectionSocketFactory sslSocketFactory = SSLConnectionSocketFactory.getSystemSocketFactory();
```

Comment:
Why create this only to possibly replace it in the next block? Should just be in an 'else' block.